### DISCUSSION TITLE
2006/08/15/talking-about-%E6%99%BA%E5%8A%9B%E9%A2%98%E4%B8%80%E5%88%99

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#107] [Announcements]
# 2006/08/15/talking-about-%E6%99%BA%E5%8A%9B%E9%A2%98%E4%B8%80%E5%88%99

这样行不行啊：

https://blog.qizhen.xyz/2006/08/15/talking-about-%E6%99%BA%E5%8A%9B%E9%A2%98%E4%B8%80%E5%88%99

**Comment by ruanqizhen** [comment]
之前博客下的留言：

fx
2006年08月15日 18:12 编辑
嗯,挺妙哈,你是商业间谍吗

回复
Qizhen
2006年08月15日 13:17 编辑
非常有意义的，没看出来吧 :)这其实是一个简化了的信息安全问题，它的推广现在已经无处不用了。

回复
Wu
2006年08月15日 12:52 编辑
纯粹从研究的角度看是可行的。不过现实中很少遇到这样的问题。没什么意义。呵呵～

回复
